docs/how-to-guides/javascript/README.md

```diff
@@ -1,6 +1,6 @@
-# Getting Started with JavaScript
+# How to use JavaScript with Gutenberg 
 
-The purpose of this tutorial is to step through getting started with JavaScript and WordPress, specifically around the new block editor. The Block Editor Handbook contains information on the APIs available for working with this new setup. The goal of this tutorial is to get you comfortable on how to use the API reference and snippets of code found within.
+The Block Editor Handbook contains information on the APIs available for working with this new setup. The goal of this tutorial is to get you comfortable on how to use the API reference and snippets of code found within.
```

Comment:
should it be `comfortable _with_ how to use the API` ?

Reply:
how about "comfortable with using "